Views/Home/Index.cshtml

```diff
@@ -111,91 +114,91 @@
-
-                            console.log('Data updated successfully.', data);
-                            // Close the modal
-                            $("[data-dismiss=modal]").trigger({ type: "click" });
-                        } else {
-                            console.error('Error updating data.', data);
-                        }
-                    },
-                    error: function (xhr, status, error) {
-                        console.error('Error updating data: ' + error);
-                    }
-                });
-            });
+            // $('.edit-button').on('click', function () {
+
+            //     editedRow = $('#elementTable').DataTable().row($(this).closest('tr'));
+            //     var projectId = $(this).data('id');
+            //     var projectName = $(this).data('name');
+            //     var projectVersion = $(this).data('version');
+
+            //     $('#editId').val(projectId);
+            //     $('#editName').val(projectName);
+            //     $('#editVersion').val(projectVersion);
+
+            // });
+
+            // $('#saveButton').on('click', function () {
+            //     var projectId = $('#editId').val();
+            //     var editedName = $('#editName').val();
+            //     var editedVersion = $('#editVersion').val();
+
+            //     var updatedData = {
+            //         id: projectId,
+            //         name: $('#editName').val(),
+            //         version: $('#editVersion').val()
+            //     };
+
+            //     console.log(updatedData);
+
+            //     $.ajax({
+            //         url: '/Home/Edit/',
+            //         type: 'POST',
+            //         data: JSON.stringify(updatedData),
+            //         contentType: 'application/json',
+            //         success: function (data) {
+            //             if(data.success) {
+            //                 var rowIndex = editedRow.index(); // Retrieve the index of the modified line
+
+            //                 table.cell(rowIndex, 0).node().querySelector('.name-field').innerText = data.editedProject.name; //Column 0: Name
+            //                 table.cell(rowIndex, 0).node().querySelector('.version-field').innerText = data.editedProject.version; // Column 0 : Version
+
+            //                 console.log('Data updated successfully.', data);
+            //                 // Close the modal
+            //                 $("[data-dismiss=modal]").trigger({ type: "click" });
+            //             } else {
+            //                 console.error('Error updating data.', data);
+            //             }
+            //         },
+            //         error: function (xhr, status, error) {
+            //             console.error('Error updating data: ' + error);
+            //         }
+            //     });
+            // });
```

Comment:
Why is this section commented out? Looks like test code, can it be removed completely?

Reply:
Here is the function to edit a project but it is not anymore usefull at the moment but I wanted to keep it for future implementations. I can also remove it.